### PR TITLE
AC-924 : Used ViewBinding in SyncedPatients package.

### PR DESCRIPTION
**Pull request by LuGO0**
## Description of what I changed

1. Used ViewBinding in SyncedPatients package.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-924

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsActivity.java [hunk 1]
```diff
@@ -27,21 +27,24 @@
...
+import org.openmrs.mobile.databinding.ActivityFindPatientsBinding;
 import org.openmrs.mobile.utilities.ApplicationConstants;
 import org.openmrs.mobile.utilities.StringUtils;
 
 public class SyncedPatientsActivity extends ACBaseActivity {
-    public SyncedPatientsPresenter mPresenter;
+    public SyncedPatientsPresenter presenter;
     private SearchView searchView;
     private String query;
     //Menu Items
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsActivity.java:38]: Can you remove this comment too

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsFragment.java [hunk 2]
```diff
@@ -46,41 +47,40 @@
 
 public class SyncedPatientsFragment extends ACBaseFragment<SyncedPatientsContract.Presenter> implements SyncedPatientsContract.View {
     // Fragment components
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsFragment.java:49]: Remove this

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsFragment.java [hunk 3]
```diff
@@ -46,41 +47,40 @@
...
-    private TextView mEmptyList;
-    private RecyclerView mSyncedPatientRecyclerView;
+    private FragmentSyncedPatientsBinding binding = null;
+    private TextView emptyList;
+    private RecyclerView syncedPatientRecyclerView;
     private SwipeRefreshLayout swipeRefreshLayout;
     //Initialization Progress bar
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsFragment.java:54]: This too. In fact remove all unnecessary comments

**Review by rishabh-997 — CHANGES_REQUESTED** [review]
Rest LGTM

**Comment by LuGO0** [comment]
@rishabh-997  made required changes please review.

**Comment by LuGO0** [comment]
@rishabh-997 made required changes please review

**Pull request merged**